tests/infrastructure/vhostmd/test_downwardmetrics_virtio.py

```diff
@@ -91,6 +92,26 @@ def parsed_metrics_command_data(vm):
     )
 
 
+def get_preference_name(rhel_version):
+    """Generate a preference name for a given RHEL version, adjusted by architecture.
+
+    Args:
+        rhel_version (str): The RHEL version (e.g., 'rhel8', 'rhel10').
+
+    Returns:
+        str: The formatted preference name (e.g., 'rhel.8', 'rhel.10.s390x', 'rhel.10.arm64').
+             For s390x or arm64 architectures, appends the architecture suffix (e.g., '.s390x')
+             unless the version is 'rhel8'. For other architectures or 'rhel8', returns the version
+             with a dot separator (e.g., 'rhel.8', 'rhel.10').
+    """
+    preference_name = re.sub(r"(\D+)(\d+)", r"\1.\2", rhel_version)
+    # Append architecture suffix for s390x or arm64, except for rhel8
+    arch = get_cluster_architecture()
+    if rhel_version != "rhel8" and arch in ("s390x", "arm64"):
```

Comment:
save and reuse in utilities constants because we need it in tests/global_config.py